pallets/schemas/src/tests.rs

```diff
@@ -350,3 +350,27 @@ fn dsnp_broadcast() {
 	);
 	assert_ok!(result);
 }
+
+#[test]
+fn create_schema_with_settings_should_work() {
```

Comment:
Error tests?

Reply:
Since schema dont directly check settings but consumers of schema does, I have added more tests and negative tests for SchemaSetting in stateful pallet